<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownLoadManager</class>
 <widget class="QDialog" name="DownLoadManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownLoadManager</string>
  </property>
  <widget class="QTextEdit" name="txtUrl">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>30</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckMonClipborad">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>7</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Monitor Clipborad</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbpath">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Download Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPath">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>357</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckAskWhenSave">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>384</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ask ervery time</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtFeedback">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="maximumBlockCount">
    <number>10000</number>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnSart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>28</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtnAxel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>57</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Axel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtnWget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>31</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>wGet</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
